### ISSUE TITLE
Going back into game from XMB causes buttons to be pressed in-game

### ISSUE THREAD

**Issue by anothername99** [libretro/RetroArch#7489]
This happens with several cores and I believe that it's a problem in RetroArch.

To reproduce, open up Quick Menu when you are in the game. Then go back into the game.

px68k: L2 gets pressed causing the internal menu to show up.

GenesisPlusGX: Start button on player 2 gets pressed. This completely breaks single player in Toejam and Earl - Panic on Funkotron.

Reicast: A+B+X+Y+Start gets pressed, triggering a soft reset. This is a bit random and doesn't happen every time.

I am running Windows Server 2016.

**Comment by orbea** [comment]
@anothername99 Is this a problem still? What RetroArch version? Have you tried a nightly?

**Comment by orbea** [comment]
I tried GenesisPlusGX with Sonic 3 and was unable to reproduce this with the udev or x input drivers using linux.

What kind of gamepad do you have? Can you confirm your controls are not mapped to any overlapping buttons? What device is set in the core's control options? Do any of the input or joypad drivers work better or worse?

**Comment by anothername99** [comment]
My gamepad is the Logitech Gamepad F310. I did some testing again with GenesisPlusGX and Toejam and Earl Panic on Funkotron, with a fresh RetroArch install and saw this:

Gamepad set to XInput mode and RetroArch's joypad driver set to xinput: Issue still the same as I described above.

Gamepad set to XInput mode and RetroArch's joypad driver set to dinput: When I pressed the menu button, besides the previous issue, RetroArch also went into slow mode, RetroArch's OSD displayed "null" and it also created a couple of 0 KB video files: https://i.imgur.com/Wa9jcOY.png

I also did a test in Linux with RetroArch's joypad driver set to udev. Here there is no issue.

When I stick to using the keyboard (and pressing F1 to go into Quick Menu and back) there is also no issue.
